Description: Revert upstream change to not use timed login after autologin session
Bug: https://bugs.edge.launchpad.net/ubuntu/+source/gdm/+bug/396489

Index: gdm-2.30.4/daemon/gdm-static-display.c
===================================================================
--- gdm-2.30.4.orig/daemon/gdm-static-display.c	2010-06-16 18:32:38.000000000 +0200
+++ gdm-2.30.4/daemon/gdm-static-display.c	2010-06-30 14:49:28.000000000 +0200
@@ -121,6 +121,9 @@
 {
         g_return_val_if_fail (GDM_IS_DISPLAY (display), FALSE);
 
+        /* Only allow timed/automatic login once */
+        GDM_STATIC_DISPLAY (display)->priv->enable_timed_login = FALSE;
+
         GDM_DISPLAY_CLASS (gdm_static_display_parent_class)->unmanage (display);
 
         return TRUE;
